examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -139,42 +131,57 @@ echo "submitting script to do compilation"
 sed -i s/\<NAME\>/standalone/g compile.daint.slurm
 sed -i s/\<NTASKS\>/$ranks/g compile.daint.slurm
 sed -i s/\<NTASKSPERNODE\>/1/g compile.daint.slurm
-sed -i s/\<CPUSPERTASK\>/$nthreads/g compile.daint.slurm
+sed -i s/\<CPUSPERTASK\>/$NTHREADS/g compile.daint.slurm
 sed -i s/--output=\<OUTFILE\>/--hint=nomultithread/g compile.daint.slurm
 sed -i s/00:45:00/03:30:00/g compile.daint.slurm
 sed -i s/cscsci/normal/g compile.daint.slurm
 sed -i s/\<G2G\>/export\ CRAY_CUDA_MPS=1/g compile.daint.slurm
 sed -i "s#<CMD>#export PYTHONOPTIMIZE=TRUE\nexport PYTHONPATH=/project/s1053/install/serialbox2_master/gnu/python:\$PYTHONPATH\nsrun python examples/standalone/runfile/dynamics.py $data_path 1 $backend $githash --disable_halo_exchange#g" compile.daint.slurm
-set +e
 # execute on a gpu node
+rm -f slurm-*.out
+set +e
 res=$(sbatch -W -C gpu compile.daint.slurm 2>&1)
-wait
+status1=$?
+grep -q SUCCESS slurm-*.out
+status2=$?
 set -e
-cleanupFailedJob "${res}"
-
-echo "DONE WAITING ${$?}"
-echo "compilation step finished"
+wait
+echo "DONE WAITING ${status1} ${status2}"
+if [ $status1 -ne 0 -o $status2 -ne 0 ] ; then
+    cleanupFailedJob "${res}"
+    echo "ERROR: compilation step failed"
+    exit 1
+else
+    echo "compilation step finished"
+fi
+mv -f slurm-*.out compile.daint.out
```

Comment:
should we just sed the output to this already?

Reply:
Agreed. Done.